.eslintrc.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
     rules: {
         // rules we've always adhered to or now do
         "max-len": ["error", {
-            code: 90,
+            code: 120,
```

Comment:
I'm tired of constantly breaking lines in weird ways in the js-sdk just to appease the length limit, and we use 120 for the react-sdk.

Reply:
I think I suggested this before, but people seemed fond of the 90 character limit.

Reply:
Personally, I prefer 90 as a max since it's easier to read two side by side files that way, but that's just my opinion.

Maybe we should discuss it as a team first, and then make a separate PR for it if we want to change?

Reply:
I'll back out the change and save my protest for another day